src/Migration/Sources/Firebase.php

```diff
@@ -213,18 +213,26 @@ private function exportUsers(int $batchSize)
             $nextPageToken = $response['nextPageToken'] ?? null;
 
             foreach ($result as $user) {
-                $users[] = new User(
+                $transferUser = new User(
                     $user['localId'] ?? '',
-                    $user['email'] ?? '',
-                    $user['displayName'] ?? $user['email'] ?? '',
-                    new Hash($user['passwordHash'] ?? '', $user['salt'] ?? '', Hash::ALGORITHM_SCRYPT_MODIFIED, $hashConfig['saltSeparator'] ?? '', $hashConfig['signerKey'] ?? ''),
-                    $user['phoneNumber'] ?? '',
+                    $user['email'] ?? null,
+                    $user['displayName'] ?? $user['email'] ?? null,
+                    null,
+                    $user['phoneNumber'] ?? null,
```

Comment:
We change the default phone number from empty string to null here, is that expected?

Reply:
The classes were updated to accept null values on the Destinations side, this allows us to say that the attribute doesn't exist at all so it won't try to add it on the Appwrite/Local adapter end